OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -533,4 +552,4 @@ public DiscussionResponseViewHolder(View itemView) {
             discussionReportViewHolder = new DiscussionReportViewHolder(itemView);
         }
     }
-}
+}
```

Comment:
Github likes an extra new line at the end of files.

Reply:
Argh apologies for missing that one